tutorial/ch00_main/main.c

```diff
@@ -63,6 +63,7 @@
 
 #include <TI/tivx.h>
 #include <vx_tutorial.h>
+#include "../../../vision_apps/utils/app_init/include/app_init.h"
```

Comment:
Please exclude the changes in the tutorial, we have fixed this in our latest

Reply:
to be removed from the PR @rb